src/service/FileTransferService.ts

```diff
@@ -0,0 +1,45 @@
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
+import FormData from 'form-data';
+import { CREATED } from 'http-status-codes';
+
+export class FileTransferService {
+
+    constructor(private readonly host: string, private readonly key: string) {
+        this.host = host;
+        this.key = key;
+    }
+
+    private uri: string = `${this.host}/dev/files`;
```

Comment:
Does this work? Or does it have to like:
`
private uri: string;
`
`
constructor(private readonly host: string, private readonly key: string) {
        this.host = host;
        this.key = key;
        this.uri = '${this.host}/dev/files';
    }`